abz/acousticbrainz.py

```diff
@@ -164,3 +164,185 @@ def process(path):
 def cleanup():
     if os.path.isfile(config.settings["profile_file"]):
         os.unlink(config.settings["profile_file"])
+        
+def generate_dataset(path):
+    """Process a direcotry containing a datase
+    
+    Args:
+        path: the relative or absolute path of the dataser rootdir
+    """
+    if not os.path.exists(path):
+        sys.exit(path + " does not exists!")
+    if not os.path.isdir(path):
+        sys.exit(path + " is not a directory!")
+    datasetdict = scan_dir(os.path.abspath(path))
+    print(datasetdict)
```

Comment:
even if this is a work in progress, it'd be nice to not have too many debug statements like this. If you need logging, consider using the logging module.

Also, pay attention to the parentheses `()` - in python2 these aren't needed.